src/gui/topbar_subwindows/politics_subwindows/gui_reforms_window.hpp

```diff
@@ -100,6 +100,15 @@ void reform_rules_description(sys::state& state, text::columnar_layout& contents
 void reform_description(sys::state& state, text::columnar_layout& contents, dcon::issue_option_id ref) {
 	auto reform = fatten(state.world, ref);
 
+	text::add_line(state, contents, reform.get_name());
```

Comment:
you need to handle the case where no name has been set (here and elsewhere) or you may mess up text for the base game

Reply:
In particular, you are always forcing a ..._desc key to be created, so that will show up for most mods

Reply:
basegame does define the name for it tho